core/Cargo.toml

```diff
@@ -300,12 +300,15 @@ suppaftp = { version = "5.2", default-features = false, features = [
   "async-rustls",
 ], optional = true }
 tikv-client = { version = "0.3.0", optional = true, default-features = false }
-tokio = "1.27"
+tokio = { version= "1.27",features = ["sync"] }
```

Comment:
Just a tiny flaw: a missing space between `,` and `features`?

Reply:
Yes🥰. Let's fix it in next PR.